City-of-Helsinki/parking-permits
# Bypass traficom setting

## Description

Flag to bypass Traficom when fetching vehicle details (should be set in admin UI, TBD).

A further change is to set the `Vehicle.updated_from_traficom_on` timestamp to NULL, as the vehicle should only set this value when the vehicle is actually fetched from traficom.

The bypass traficom flag should be reset to `False` when the user changes their vehicle.

## Context




[PV-690](https://helsinkisolutionoffice.atlassian.net/browse/PV-690)

## How Has This Been Tested?

Unit tests and manual testing with Webshop

## Manual Testing Instructions for Reviewers

Create a permit as usual in Webshop, and in Django admin for the Parking Permit set Bypass Traficom validation checkbox.

When refreshing Webshop page, no Traficom validation should be required.


[PV-690]: https://helsinkisolutionoffice.atlassian.net/browse/PV-690?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
